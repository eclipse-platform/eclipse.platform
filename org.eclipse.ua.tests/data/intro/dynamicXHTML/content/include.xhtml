<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<!--
   The include test page. This page includes a line from include2.xhtml.
-->

<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>include.xhtml</title>
</head>

<body>
<h2>include.xhtml</h2>
<p>
This page contains include examples. Below there should be one line of text that is included from
another file, include2.xhtml:
</p>
<include path="include2/my.include.id"/>
</body>
</html>
