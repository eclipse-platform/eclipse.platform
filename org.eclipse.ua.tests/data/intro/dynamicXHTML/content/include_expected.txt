<!--
   The include test page. This page includes a line from include2.xhtml.
--><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>

<title>include.xhtml</title>
</head>

<body>
<h2>include.xhtml</h2>
<p>
This page contains include examples. Below there should be one line of text that is included from
another file, include2.xhtml:
</p>
<ul id="my.include.id">
<li>This text comes from include2.xhtml and can be included by other documents.</li>
</ul>
</body>
</html>